### PR TITLE
[Fix] Add s-pipes dependencies for tests, update deprecated method

### PR DESCRIPTION
**Pull request by Matthew-Kulich**

### CODE REVIEW

**Comment by Matthew-Kulich** [comment]
I have no idea how is possible that the tests were working until now

File: s-pipes-test/src/main/java/cz/cvut/spipes/test/JenaTestUtils.java [hunk 1]
```diff
@@ -14,7 +14,7 @@ public class JenaTestUtils {
 
     public static void mapLocalSPipesDefinitionFiles() {
         OntDocumentManager dm = OntDocumentManager.getInstance();
-        dm.setFileManager(FileManager.get());
+        dm.setFileManager(FileManager.getInternal());
```
Comment (blcham) [inline s-pipes-test/src/main/java/cz/cvut/spipes/test/JenaTestUtils.java:17]: why do we need this, what is the difference between those methods?

**Review by blcham — APPROVED** [review]
LGTM, just let me know w.r.t. comments

File: s-pipes-test/src/main/java/cz/cvut/spipes/test/JenaTestUtils.java (hunk 1 above)
Comment (Matthew-Kulich) [reply s-pipes-test/src/main/java/cz/cvut/spipes/test/JenaTestUtils.java:17]: So when I see deprecated method in the file I'm modifying so I would change it to newer one

Comment (blcham) [reply s-pipes-test/src/main/java/cz/cvut/spipes/test/JenaTestUtils.java:17]: I see, it just evocate to me that we should not use getInternal() either

**Comment by blcham** [comment]
we can merge it

**Pull request merged**